draft-ietf-ppm-dap.md

```diff
@@ -1391,14 +1397,16 @@ is set to `AggregationJobInitReq` and the media type is set to
 The Leader MUST authenticate its requests to the Helper using a scheme that
 meets the requirements in {{request-authentication}}.
 
-After receiving HTTP status 201 Created, the Leader proceeds to poll the
-aggregation job by sending HTTP GET requests to
+The Helper responds with HTTP status 201 Created. If the response has a body
+containing an `AggregationJobResp`, the Leader proceeds onward (see
+{{aggregation-helper-init}}). Otherwise, if the response has an empty body, the
+Leader polls the aggregation job by sending HTTP GET requests to
```

Comment:
Instead of using empy/non-empty to distinguish the response, the Leader should use the media type. That is, if the response contains the media type header for AggregationJobResp (I don't remember what this is), then interpret the body of the response as an AggregationJobResp.